uploadNeuroDB/NeuroDB/ExitCodes.pm

```diff
@@ -105,8 +108,10 @@ our $ARG_FILE_DOES_NOT_EXIST = 5; # if file given as an argument does not exist
 # database related failure
 # called from minc_insertion.pl & register_processed_data.pl
 our $FILE_NOT_UNIQUE = 6; # if file to register is not unique & already
-                           # inserted
+                          # inserted
 
+# other generic failure
+our $FILE_OR_FOLDER_DOES_NOT_EXIST = 6; # if file or folder does not exist
```

Comment:
just curious at using exit code =6 when 2 lines before we have the same one? is this intentional?

Reply:
Oups, you're right, it should have been 7. I did not use the exit 5 as the exit 5 is specific to files passed as an *argument* that do not exist and here, the file or folder was not provided as an argument. Does that make sense?